isomdurm/project0/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
@isomdurm Could you fill out the project submission form?  This should be a separate repo; not in homework.